components/annot-age-values.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <b-card no-body>
+      <b-card-header>I am a header</b-card-header>
+      <b-card-body style="position:relative; height:300px; overflow-y:scroll;">
+        <annot-continuous-values :items="unique_table_data"></annot-continuous-values>
+      </b-card-body>
+    </b-card>
+    <b-button variant="success" @click="applyTransform">Confirm and Upload</b-button>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: "sub-numeric-validation",
+  data() {
+    return {
+      // TODO turn the range into an argument for the component
+      unique_range: {start: 0, end: 3},
+      regularExpressions: [ // using named capture groups
```

Comment:
It's best to heavily document regular expressions when they are created. This way when you or someone new revisits the regex code, all pieces of the expression are immediately available without visual parsing of them.